tests/session/cmd/test_sequential.py

```diff
@@ -88,15 +88,15 @@ def test_result_json_sequential(
     packaging_test = get_cmd_exit_run_id(log_report, ".pkg", "test")
     assert packaging_test == [(None, "build_wheel")]
     packaging_installed = log_report["testenvs"][".pkg"].pop("installed_packages")
-    assert {i[: i.find("==")] for i in packaging_installed} == {"pip", "setuptools", "wheel"}
+    assert {i.partition("==")[0] for i in packaging_installed} == {"pip", "setuptools", "wheel"}
```

Comment:
In these two cases here I find `find` easier to read and understand than partition 🤔

Reply:
First, I experimented with `assert all(i.startswith(("pip==", "setuptools==", "wheel==")) for i in packaging_installed)` but `pytest` would probably won't be very helpful if the test failed. I'll revert my change.